stumpy/core.py

```diff
@@ -4210,3 +4206,71 @@ def _mpdist(
     MPdist = _select_P_ABBA_value(P_ABBA, k, custom_func)
 
     return MPdist
+
+
+def process_isconstant(T, m, T_subseq_isconstant, T_subseq_isfinite=None):
+    """
+    Compute the rolling isconstant for 1-D and 2-D arrays.
```

Comment:
I think this should start off by saying:

"This is a convenience wrapper around the `rolling_isconstant` and `fix_isconstant_isfinite_conflicts`

Reply:
Good point! I added this sentence at the top of the docstring.